tests/misc.spec.ts

```diff
@@ -47,6 +48,74 @@ describe("Misc tests", function () {
     datas[0].body.toString().should.equal(msgString);
   });
 
+  it("should be able to send and receive a JSON object as a message correctly", async function () {
+    const partitionId = hubInfo.partitionIds[0];
+    const msgBody = {
+      id: '123-456-789',
+      weight: 10,
+      isBlue: true,
+      siblings: [
+        {
+          id: '098-789-564',
+          weight: 20,
+          isBlue: false,
+        }
+      ]
+    };
+    const obj: EventData = { body: msgBody };
+    debug("Sending one message %O", obj);
+    breceiver = BatchingReceiver.create((client as any)._context, partitionId, { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
+    await client.send(obj, partitionId);
+    debug("Successfully sent the large message.");
+    const datas = await breceiver.receive(5, 5);
```

Comment:
>datas [](start = 10, length = 5)

nitpick level: > 9000: data is already plural.

Reply:
🥇